packages/react/src/components/Embed/Embed.tsx

```diff
@@ -131,8 +131,8 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
         className={classes.root}
         onClick={this.handleClick}
         {...accessibility.attributes.root}
-        {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
         {...unhandledProps}
+        {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
```

Comment:
This is correct `applyAccessibilityKeyHandlers` should always be spread after the `unhandledProps`, as it will invoke the key events provided by the behavior.

Reply:
yes, there's now even a test verifying that for all compoennts (that have the actions defined)